lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -3460,25 +3460,39 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
 
     describe "TRANS_NEW_CREATE_MINT_SCRIPTS - I can mint and burn with different policy scripts" $ do
         let scenarios =
-                  [ ( "all", [json|{ "all": [ "cosigner#0" ] }|] )
-                  , ( "any", [json|{ "any": [ "cosigner#0" ] }|] )
-                  , ( "some", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0" ]} }|] )
-                  , ( "all, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 } ] }|] )
-                  , ( "some, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_until": 57297561 } ]} }|] )
-                  , ( "all, active_from 10", [json|{ "all": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "any, active_from 10", [json|{ "any": [ "cosigner#0",  { "active_from": 10 } ] }|] )
-                  , ( "some, active_from 10", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 } ]} }|] )
-                  , ( "all, active_from 10, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
-                  , ( "any, active_from 10, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] )
-                  , ( "some, active_from 10, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 }, { "active_until": 57297561 } ]} }|] )
+                  [ ( "all", [json|{ "all": [ "cosigner#0" ] }|], False )
+                  , ( "any", [json|{ "any": [ "cosigner#0" ] }|], False )
+                  , ( "some", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0" ]} }|], False )
+                  , ( "all, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_until": 57297561 } ] }|], False )
+                  , ( "any, active_until 57297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 } ] }|], False )
+                  , ( "some, active_until 57297561", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_until": 57297561 } ]} }|], False )
+                  , ( "all, active_from 10", [json|{ "all": [ "cosigner#0",  { "active_from": 10 } ] }|], True )
+                  , ( "any, active_from 10", [json|{ "any": [ "cosigner#0",  { "active_from": 10 } ] }|], True )
+                  , ( "some, active_from 10", [json|{ "some": {"at_least": 1, "from": [ "cosigner#0", { "active_from": 10 } ]} }|], True )
+                  , ( "all, active_from 10, active_until 57297561", [json|{ "all": [ "cosigner#0",  { "active_from": 10 }, { "active_until": 57297561 } ] }|] , True)
+                  , ( "any, active_until 57297561, active_from 58297561", [json|{ "any": [ "cosigner#0",  { "active_until": 57297561 }, { "active_from": 58297561 } ] }|] , False)
```

Comment:
This case works, but if I modify it to:
```
{ "any": [ "cosigner#0",  { "active_until": 58297561 }, { "active_from": 10 } ] }
```
It gives:
```
 {"code":"created_wrong_policy_script_template",
  "message":"It looks like I've created a transaction with a minting/burning policy script that either does not pass validation, contains more than one cosigner, or has a cosigner that is different from cosigner#0."}
```

bug?

Reply:
Ok, read this now:

>this was illegal script ReduntantTimelocks https://github.com/input-output-hk/cardano-addresses/blob/master/core/lib/Cardano/Address/Script.hs#L405

But I don't quite understand why it is illegal... :thinking:

Reply:
Ok, read this now:

>this was illegal script ReduntantTimelocks https://github.com/input-output-hk/cardano-addresses/blob/master/core/lib/Cardano/Address/Script.hs#L405

But I don't quite understand why it is illegal... :thinking:

Reply:
I don't know, we have add it with Matthias long time ago https://github.com/input-output-hk/cardano-addresses/blob/master/core/lib/Cardano/Address/Script.hs#L436

Reply:
ok, now I remember, see if you have `any [from 10, until 100]` then it is always satisfied because in every slot you are satisfying one of it. so why to specify it at all? Hence it is outlawed as redundant.